src/App.js

```diff
@@ -11,7 +19,37 @@ export default class App extends React.Component {
 
   render() {
     return (
-    <Calculator />
+    <Router>
+      <header>
+        <h1>
+          MathMagicians
+        </h1>
+        <nav>
+          <ul>
+            <li>
+              <Link to="/" className="link">Home</Link>
+            </li>
+            <li>
+              <Link to="/calculator" className="link">Calculator</Link>
+            </li>
+            <li>
+              <Link to="/quote" className="link">Quote</Link>
+            </li>
+          </ul>
+        </nav>
+        </header>
+      <Switch>
+        <Route path="/calculator">
+          <Calculator />
+        </Route>
```

Comment:
- [ ] There is a small issue when I try to divide a number by zero, let us say `5/0` normally it has to display an `error` in the screen but currently it is showing this error
![e](https://user-images.githubusercontent.com/44745855/129792521-d97b2ced-e38a-48d5-8673-ac91db512932.png)

KIndly fix that and you will be fine.